src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -58,6 +62,11 @@ namespace Orleans.Runtime
     /// </summary>
     public class Silo : MarshalByRefObject // for hosting multiple silos in app domains of the same process
     {
+        public override object InitializeLifetimeService()
```

Comment:
It's for dependency injection, @attilah may be able to shed more light. Not sure why it's in the diff